draft-ietf-scitt-software-use-cases.md

```diff
@@ -120,7 +120,7 @@ Recently, the global community has experienced numerous supply chain attacks tar
 ~~~~
 {: #lifecycle-threats title="Example Lifecycle Threats"}
 
-DevSecOps often depends on third-party and open-source solutions.
+DevSecOps often depends on third-party and open-source software.
```

Comment:
Minor, non-blocking: I do not think this is controversial, but I guess I will pose that open source software and library can be leveraged into making a compositional solution, but independent open source libraries and software applications cannot compromise a full solution as supplied. Even for AGPL'ed services, technically a web application released as open-source software, the separate service deployed on an application is a solution. My wordy comments on minor word changes like this, believe it or not, affirm my belief how this is going to be an important. You can build whole demos on this word choice changing back and forth because then you could actually equate the former with the latter once you verify it. 😄